app/src/ui-blokada/res/values/strings_logger.xml

```diff
@@ -4,6 +4,7 @@
     <string name="logger_slot_title">Request logger</string>
     <string name="logger_slot_desc">When enabled, Blokada will create a CSV file and log requests that have been allowed and / or blocked by the app. Look for "blokada" folder on your external storage.</string>
```

Comment:
Description doesn't match the new functionality.

Reply:
Done.